src/SFA.DAS.EAS.Employer_Financial.Database/Scripts/PreDeployment/AML-2671-PopulateNullApprenticeshipCourseNames.sql

```diff
@@ -0,0 +1,50 @@
+﻿BEGIN TRY
+BEGIN TRANSACTION
```

Comment:
It doesn't make much of a difference to the execution of the SQL but the BEGIN TRAN is within the BEGIN TRY but the COMMIT TRAN is outside, which makes the scope of the TRY and TRAN overlap slightly. It's usually better to have purely nested scopes (this is generally true in code - cognitively it makes it easier to deal with). To achieve this L49-50 should be moved to L34 (and we don't need L49 anymore, as tran count will always be 1 at that point).

Reply:
Suggested changes made.